server/routes/visit.test.ts

```diff
@@ -31,6 +34,16 @@ const visitSessionsService = new VisitSessionsService(
 const auditService = new AuditService() as jest.Mocked<AuditService>
 const prisonerVisitorsService = new PrisonerVisitorsService(null, systemToken) as jest.Mocked<PrisonerVisitorsService>
 
+jest.mock('./visitorUtils', () => {
+  const visitorUtils = jest.requireActual('./visitorUtils')
+  return {
+    ...visitorUtils,
+    clearSession: jest.fn((req: Express.Request) => {
```

Comment:
For testing, don't actually clear the session so that an object can be passed through and populated